testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/interceptor/register/TestSLSB.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2018, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 2110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.jboss.as.test.integration.ejb.interceptor.register;
+
+import javax.ejb.Stateless;
+import java.util.logging.Logger;
+
+@Stateless
+//@LocalBean
+public class TestSLSB implements TestRemote {
+
+  private Logger log = Logger.getLogger(TestSLSB.class.getName());
+
+  public void invoke() {
+    log.info("Testing string: SLSB successfully invoked");
```

Comment:
Test should not log at info, as it generates IO ops which slow down the servers. Any logging that is generated as part of normal test execution should be debug or trace so it is not logged by default.